specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2020-08-15-preview/HybridCompute.json

```diff
@@ -0,0 +1,1205 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "HybridComputeManagementClient",
+    "description": "The Hybrid Compute Management Client.",
+    "version": "2020-08-15-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HybridCompute/machines/{name}": {
+      "put": {
+        "tags": [
+          "machines"
+        ],
+        "operationId": "Machines_CreateOrUpdate",
+        "description": "The operation to create or update a hybrid machine resource identity in Azure.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
```

Comment:
any reasons we go common-types for ApiVersionParameter and SubscriptionParameter again? Our previous had defined them locally.

Reply:
the previous base has them as common-types, so I updated mine to use the common-types. the tool is complaining as duplicates if one file uses common-types and other the local ones. so made it uniform